ls1intum/Artemis
# `Development`: Improve server test coverage and code quality

I added additional server tests and improved the general server code quality and consistency.

### Checklist
#### General

### Steps for Testing

Prerequisites:
- An Instructor

1. Create a new programming exercise
2. Import a programming exercise
3. Make sure everything works correctly

#### Code Review
#### Manual Tests

### Test coverage
Server test coverage is finally higher than 90% 🥇
